xOpera-rest-blueprint/modules/openstack/playbooks/vm_create.yml

```diff
@@ -18,22 +16,22 @@
     - name: Set attributes
       set_stats:
         data:
-          private_address:  "{{ server_info.server.private_v4 }}"
-          public_address:   "{{ server_info.server.public_v4 }}"
-          id:               "{{ server_info.server.id }}"
-
+          private_address: "{{ server_info.server.private_v4 }}"
+          public_address: "{{ server_info.server.public_v4 }}"
+          id: "{{ server_info.server.id }}"
 
     - name: Create temporary invetory for ssh wait
       add_host:
         name: server
         groups: vms
         ansible_host: "{{ server_info.server.public_v4 }}"
-        ansible_user: centos
```

Comment:
I would leave this IdentityFile since we should give its location through inputs - we don't have the ssh config set here..